dojo/product/views.py

```diff
@@ -1160,6 +1160,9 @@ def edit_meta_data(request, pid):
 @user_is_authorized(Product, Permissions.Finding_Add, 'pid', 'staff')
 def ad_hoc_finding(request, pid):
     prod = Product.objects.get(id=pid)
+    test_type, created = Test_Type.objects.get_or_create(name="Pen Test")
+    if created:
+        test_type.save()
```

Comment:
Saving is not necessary with `get_or_create`

Reply:
I just moved the code but I can refactor it this way.

Reply:
Fixed